dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/aspect/AccessLogAspect.java

```diff
@@ -99,11 +101,11 @@ private String parseArgs(ProceedingJoinPoint proceedingJoinPoint, AccessLogAnnot
         if (annotation.ignoreRequestArgs().length > 0) {
             String[] parameterNames = ((MethodSignature) proceedingJoinPoint.getSignature()).getParameterNames();
             if (parameterNames.length > 0) {
-                List<String> ignoreList = Arrays.stream(annotation.ignoreRequestArgs()).collect(Collectors.toList());
+                Set<String> ignoreSet = new HashSet<>(Arrays.stream(annotation.ignoreRequestArgs()).collect(Collectors.toList()));
```

Comment:
```suggestion
Set<String> ignoreSet = Arrays.stream(annotation.ignoreRequestArgs()).collect(Collectors.toSet());
```

Reply:
Yeah. The HashSet object can be constructed by toSet() directly.